### PR TITLE
🌐 Add Chinese translation for Tutorial - Response Model

### PR DESCRIPTION
**Pull request by waynerv**
@RunningIkkyu @Dustyposa Please take a look.

### CODE REVIEW

File: docs/zh/docs/tutorial/response-model.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+!!! info
+    你还可以使用：
+
+    * `response_model_exclude_defaults=True`
+    * `response_model_exclude_none=True`
+
+    参考 <a href="https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict" class="external-link" target="_blank">Pydantic 文档</a> 中对 `exclude_defaults` 和 `exclude_none` 的描述。
+
+#### 默认值字段具有实际值的数据
```
Comment (maoyibo) [inline docs/zh/docs/tutorial/response-model.md:135]: ### 与默认值相同的数据

File: docs/zh/docs/tutorial/response-model.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    "name": "Bar",
+    "description": "The bartenders",
+    "price": 62,
+    "tax": 20.2
+}
+```
+
+这些值将包含在响应中。
+
+#### 具有与默认值相同值的数据
```
Comment (maoyibo) [inline docs/zh/docs/tutorial/response-model.md:150]: 与默认值相同的数据

File: docs/zh/docs/tutorial/response-model.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+!!! tip
+    请注意默认值可以是任何值，而不仅是`None`。
+
+    它们可以是一个列表（`[]`），一个值为 `10.5`的 `float`，等等。
+
+### `response_model_include` 和 `response_model_exclude`
+
+你还可以使用*路径操作装饰器*的 `response_model_include` 和 `response_model_exclude` 参数。
+
+它们接收一个由属性名称组成的 `set` 或者 `list` 来包括（但忽略其余的）或者排除（但包括其余的）指定属性。
```
Comment (maoyibo) [inline docs/zh/docs/tutorial/response-model.md:177]: 他们接收一个包含属性名称 `str` 的 `set`，来包含（省略其它的）或排除（包含其他的）它们。

**Review by maoyibo — COMMENTED** [review]
good job.

File: docs/zh/docs/tutorial/response-model.md (hunk 1 above)
Comment (waynerv) [reply docs/zh/docs/tutorial/response-model.md:135]: In context, my translation is more accurate.

File: docs/zh/docs/tutorial/response-model.md (hunk 2 above)
Comment (waynerv) [reply docs/zh/docs/tutorial/response-model.md:150]: Same as above.

File: docs/zh/docs/tutorial/response-model.md (hunk 1 above)
Comment (maoyibo) [reply docs/zh/docs/tutorial/response-model.md]: How about 数据具有与默认值相同的实际值, just to make it easier to understand.

**Review by Laineyzhang55 — APPROVED** [review]

**Review by yanever — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks a lot @waynerv ! :rocket: :coffee: 

And thanks for the reviews @yanever, @Laineyzhang55, and @maoyibo :bow: :cake:

**Pull request merged**